src/sagemaker/clarify.py

```diff
@@ -1399,6 +1580,45 @@ def get_explainability_config(self):
         return copy.deepcopy({"shap": self.shap_config})
 
 
+class AsymmetricSHAPConfig(ExplainabilityConfig):
+    """Config class for Asymmetric SHAP algorithm for TimeSeries explainability"""
+
+    def __init__(
+        self,
+        explanation_type: str = ASYM_SHAP_DEFAULT_EXPLANATION_TYPE,
```

Comment:
Instead of str you can use `typing.Literal`
```
explanation_type: Literal["fine_grained", "timewise_chronological"]
```

With that IDE's will suggest the names, which makes APIs more user friendly

Reply:
Will address today.